src/platform/macos/mod.rs

```diff
@@ -832,22 +824,26 @@ fn select(
 pub struct OsIpcOneShotServer {
     receiver: OsIpcReceiver,
     name: String,
+    #[allow(dead_code)]
```

Comment:
This field looks like it's read in the destructor. Why is this necessary?

Reply:
Hrm. I think I was just confused at the moment that I added this. It isn't necessary. I've pushed a new version of this change that removes it.